src/components/DiagnosticCard/DiagnosticCard.scss

```diff
@@ -1,5 +1,25 @@
-.diagnostic-card {
-    min-width: 200px;
-    max-width: 350px;
+.ydb-diagnostic-card {
+    flex-shrink: 0;
+
+    width: 206px;
```

Comment:
it seems that height should be fixed too.

Reply:
It isn't fixed because of Healthcheck preview. In some rare cases there could be a lot of issues of different types in database, in this case HC Preview will be bigger and Card should grow in terms of height. 

We won't apply height that will cover all the possible scenarios, as it doesn't look as good as the current design. It's a very rare case, but we want content to be shown normally in all the cases.